kyc_sanction_scanner/models/res_partner.py

```diff
@@ -33,7 +33,10 @@ def _get_next_ongoing_monitoring_from_response(self, response):
         if backend.tech_name == "kyc_sanction_scanner":
             if len(response) != 1 and type(response) is not dict:
                 response = response[0]
-            if response.get("Result", {}).get("NextMonitoringDate", False):
-                date = (response["Result"]["NextMonitoringDate"]).split(".")[0]
-                res = datetime.strptime(date, "%Y-%m-%dT%H:%M:%S")
+            if response.get("IsSuccess", False):
+                if response.get("Result", {}).get("NextMonitoringDate", False):
```

Comment:
combine both conditions: if contition A and condition B. There is no need for 2 if statements in this case